api-1.9/src/main/java/org/openmrs/module/atomfeed/client19/impl/AtomFeedClientImpl.java

```diff
@@ -0,0 +1,153 @@
+package org.openmrs.module.atomfeed.client19.impl;
+
+import org.ict4h.atomfeed.client.AtomFeedProperties;
+import org.ict4h.atomfeed.client.repository.AllFeeds;
+import org.ict4h.atomfeed.client.repository.jdbc.AllFailedEventsJdbcImpl;
+import org.ict4h.atomfeed.client.repository.jdbc.AllMarkersJdbcImpl;
+import org.ict4h.atomfeed.client.service.EventWorker;
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.module.atomfeed.api.exceptions.AtomfeedException;
+import org.openmrs.module.atomfeed.api19.utils.AtomfeedUtils;
+import org.openmrs.module.atomfeed.client.AtomFeedClient;
+import org.openmrs.module.atomfeed.transaction.support.AtomFeedSpringTransactionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.net.URI;
+import java.util.HashMap;
+
+@Component("atomfeed.AtomFeedClient")
+@OpenmrsProfile(openmrsPlatformVersion = "1.9.* - 2.0.0")
+@Scope("request")
+public class AtomFeedClientImpl implements AtomFeedClient {
+	private static final Logger LOGGER = LoggerFactory.getLogger(AtomFeedClient.class);
+
+	private final AtomFeedProperties atomFeedProperties;
+
+	private URI uri;
+
+	private EventWorker eventWorker;
+
+	public AtomFeedClientImpl() {
+		this(null);
+	}
+
+	public AtomFeedClientImpl(EventWorker eventWorker) {
+		atomFeedProperties =  new AtomFeedProperties();
+		this.eventWorker = eventWorker;
+	}
+
+	@Override
+	public void process() {
+		LOGGER.info("{} started processing", getClass().getName());
+		validateConfiguration();
+
+		org.ict4h.atomfeed.client.service.AtomFeedClient atomFeedClient = createAtomFeedClient();
```

Comment:
can you import this package ?

Reply:
I can't because we currently have the "org.openmrs.module.atomfeed.client.AtomFeedClient;" import statement in that class.

Reply:
ok